### ISSUE TITLE
warning

### ISSUE THREAD

**Issue by letarg0** [jeffgriffith/mnist-tensorflow-demo#1]
WARNING:tensorflow:From mnist_tf_demo.py:47: softmax_cross_entropy_with_logits (from tensorflow.python.ops.nn_ops) is deprecated and will be removed in a future version.
Instructions for updating:

Future major versions of TensorFlow will allow gradients to flow
into the labels input on backprop by default.

See tf.nn.softmax_cross_entropy_with_logits_v2.

second trouble is data on local home on mac.

**Comment by Mayur2802** [comment]
just use this "tf.nn.softmax_cross_entropy_with_logits_v2" instead of "tf.nn.softmax_cross_entropy_with_logits" in your code. This should probably remove the warning.
"v2" is the newer updated version and has some additional usability.
For further explanation visit the official documentation or visit "https://stats.stackexchange.com/questions/327348/how-is-softmax-cross-entropy-with-logits-different-from-softmax-cross-entropy-wi"
